### ISSUE TITLE
jQuery and Ajax

### ISSUE THREAD

**Issue by kih7** [haxball/haxball-issues#620]
Is it possible to use JQuery and Ajax

**Comment by thenorthstar** [comment]
> Is it possible to use JQuery and Ajax

@kih7 Yes, you can use. For example, to detect location from IP address. Of course you have to convert the player **conn** into the IP address then go.

**Comment by ghost** [comment]
> 



> > Is it possible to use JQuery and Ajax
> 
> @kih7 Yes, you can use. For example, to detect location from IP address. Of course you have to convert the player **conn** into the IP address then go.

But, how do I convert the Player conn to an ip?
